css/timers.css

```diff
@@ -6,66 +6,280 @@ body{
+    border-radius: 0px;
     margin-bottom: 10px;
     height: 75px;
-    box-shadow: 0 0 8px gray;
+    color:#784a32;
+    font-size: 30px;
+    position: relative;
+}
+#addtimerbutton {
+    background-color: #e6e6e6;
+    border-radius: 0px;
+    height: 75px;
+    margin-top: 10px;
 }
 
-.timer:not([data-state="settings"]) {
-    border-top-right-radius: 0px;
-    border-bottom-right-radius: 0px;
-    top: -85px;
-    left: -115px;
-    border: 1px black;
+#timers[data-state="settings"]{
+    overflow-y: auto;
+    overflow-x: hidden;
+    max-height: 300px;
+}
+.timer[data-state="settings"]:last-of-type{
+     margin-bottom: 0;
+ }
+
+#bottom_fade {
+    z-index: 99;
+    position: fixed;
+    height: 500px;
+    bottom: 0%;
+    background-image: url("../img/bottom-fade.png");
+}
+
+.addTimerButtonContents{
+    color:#784a32;
+    font-size: 30px;
     position: relative;
+    top: 25%;
+    left: 25%;
+}
+
+.timer[data-state="settings-selected"] {
+    background-color: #ff9a62;
+    border-radius: 0px;
+    margin-bottom: 10px;
+    height: 75px;
 }
 
 .timer[data-state="closed"] {
-    width: 75px;
-    transition: 1s ease;
+    background-color:white;
+    border-top-left-radius: 15px;
+    border-bottom-left-radius: 15px;
+    top: 93px;
+    left: -126px;
+    border: 1px black;
+    position: relative;
+    width: 125px;
+    transition: 0s ease;
+    transform: none;
+}
+.timerTitle{
+    font-size:18px;
+    margin-left:10px;
+    margin-right:10px;
+}
+.timerTime{
+    text-align: center;
+    margin-right:15px;
+    display: none;
+}
+@-webkit-keyframes completed {
+     0% {
+         background-color: red;
+         opacity:1;
+     }
+     50% {
+         background-color: white;
+     }
+     100% {
+         background-color: red;
+     }
+ }
+
+.timer[data-state="completed"]  {
+    background-color: red;
+    -webkit-animation-name: completed;
+    -webkit-animation-duration: 1000ms;
+    -webkit-animation-iteration-count: 5;
+    -webkit-animation-timing-function: ease-in-out;
+    border-top-left-radius: 15px;
+    border-bottom-left-radius: 15px;
+    top: 93px;
+    left: -126px;
+    border: 1px black;
+    position: relative;
+    width: 125px;
+    transition: 0s ease;
+    transform: none;
+}
+.timerCountdown{
+    font-size:15px;
+    text-align: center;
+    margin-right:15px;
+    display: block;
+}
+input[type='number'] {
+     -moz-appearance:textfield;
+ }
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+}
+.timerColon{
+    font-size: 18px;
+}
+.timerTimeMin{
+    font-size: 18px;
+    text-align: center;
+    width:20px;
+}
+.timerTimeSec{
+    font-size: 18px;
+    width:20px;
+}
+.addMinuteButton{
+    width:60px;
+    margin-right:15px;
+    font-size:15px;
+    display: none;
+}
+.cancelTimerButton{
+    display:none;
+    width:75px;
+    font-size:15px;
+}
+.timer[data-state="settings"] .timerTitle{
```

Comment:
See my comment about cancelTimerButton. it'd be easier to make changes to this in the future since it wouldn't be duplicated for each of the states. So overall, apply these changes to `.timerTitle` `.timerTime` and `.cancelTimerButton` and any others that I might not have noticed :last_quarter_moon_with_face: